app/controllers/sidebar_sections_controller.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+class SidebarSectionsController < ::ApplicationController
+  requires_login
+
+  def create
+    sidebar_section =
+      SidebarSection.create!(
+        section_params.merge(user: current_user, sidebar_urls_attributes: links_params),
+      )
+
+    render json: SidebarSectionSerializer.new(sidebar_section)
+  rescue ActiveRecord::RecordInvalid => e
+    render_json_error(e.record.errors.full_messages.first)
+  end
+
+  def update
+    sidebar_section = SidebarSection.find(section_params["id"])
+    @guardian.ensure_can_edit!(sidebar_section)
```

Comment:
```suggestion
    sidebar_section = SidebarSection.find_by(id: section_params["id"], user_id: current_user.id)
```

I believe we can just scope the search here to the current user and avoid having to do a guardian check?

Reply:
I would like to keep guardian. In the next phase, admin will be able to create global sections. I was planning to use the same controller and guardian to protect access

Reply:
:+1: Thank you for clarifying.